health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -367,14 +370,14 @@ const ViewHierarchy = () => {
                                             <div>
                                                 <div style={{fontWeight:"600", fontSize:"1.2rem"}}>
                                                     {/* {t(hierItem?.boundaryType)} */}
-                                                    {`${t(( hierarchyType + "_" + hierItem?.boundaryType).toLowerCase().replace(/\s+/g, "_"))}`}
+                                                    {`${t(( hierarchyType + "_" + hierItem?.boundaryType).toUpperCase().replace(/\s+/g, "_"))}`}
                                                 </div>
                                                 <div style={{height:"1rem"}}></div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)